pymetamap/SubprocessBackend.py

```diff
@@ -68,19 +68,19 @@ def extract_concepts(self, sentences=None, ids=None,
 
         input_file = None
         if sentences is not None:
-            input_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = tempfile.NamedTemporaryFile(mode="wb", delete=False)
         else:
-            input_file = open(filename, 'r')
-        output_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = open(filename, 'wb')
```

Comment:
I this required for python 3.5 support? This line was suppose to load a file to process rather than creating a file from a list of strings.